client/src/CustomiseGameForm/CustomiseGameForm.css

```diff
@@ -6,19 +6,17 @@
 
 .inputFieldContainer {
 	display: flex;
-	border: 5px solid white;
-}
-
-.inputLabel {
-	border: 5px solid white;
+	align-items: center;
 }
 
+.inputFieldContainer,
```

Comment:
you mean apply a style to multiple classes.  You've been stuck on a back end project for too long.  There's even a new shorthand, say, for instance if you wanted to target all `div` elements inside a bunch of classes, you can now do `:is(.class1, .class2, .class2) div` rather than `.class1 div, .class2 div, .class3 div`